.github/workflows/ci.yml

```diff
@@ -18,28 +32,47 @@ jobs:
         uses: actions/setup-node@v1
         with:
           node-version: ${{ matrix.node-version }}
-      - name: Install
+      - name: Install dependencies
         run: |
           make setup
-        env:
-          CI: true
-      - name: Run Lint
+      - name: Linting
         run: |
           make lint
-        env:
-          CI: true
-      - name: Run Unit Tests
+
+  testing-coverage:
+    needs: setup-job
+    runs-on: ubuntu-20.04
+
+    strategy:
+      matrix:
+        node-version: [14.15.1]
+
+    steps:
+      - uses: actions/checkout@v2
+      - name: Use Node.js ${{ matrix.node-version }}
+        uses: actions/setup-node@v1
+        with:
+          node-version: ${{ matrix.node-version }}
+      - name: Install dependencies
+        run: |
+          make setup
+      - name: Test with pytest
```

Comment:
There's no pytest for UI code :)

It uses Jest.

Reply:
Fixed